README.md

```diff
@@ -1,2 +1,114 @@
-# tw-academy-s9
-This is homework for session 9
+# Sample Style Guide
+## Technical Writing Academy - May 2021
+------
+## Table of Contents
+Bold
+
+Italics
+
+Fixed Font
+
+Abbreviations
+
+Capitalization
+
+Acronyms
+
+Commas in Lists
+
+Numbers
+
+Contractions
+
+Apostrophes
+
+## **Bold**
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:
+
+● Click the Create button to start the process.
+
+● Enter your IP address in the IP Address field.
+
+## **Italics**
+
+Use italics to emphasize a word or group of words. For example:
+
+● At least two reviews are required before you check in your code.
+
+● You must not enter without your ID card.
+
+
+**Important:** Use this style sparingly to avoid diluting its strength.
+
+## **Fixed Font**
+
+Use a fixed font to highlight:
+
+● Commands or pieces of code. For example:
+
+Run the `npm install text-cli` command to install the utility.
+
+● File or folder names. For example:
+
+Read the README.md file before you proceed.
+
+## **Abbreviations**
+
+Never use the following abbreviations in a formal document:
+
+|     |   |
```

Comment:
You can use this first row of a table in Markdown to name each column. Markdown will format it as a heading row.